### ISSUE TITLE
Release v1.65.2

### ISSUE THREAD

**Issue by deckhouse-BOaTswain** [deckhouse/deckhouse#10391]
:robot: A dedicated issue to run tests and deploy release [v1.65.2](https://github.com/deckhouse/deckhouse/milestone/357).

---

<details>
<summary>Release issue commands and options</summary>
<br />

You can trigger release related actions by commenting on this issue:

- `/deploy/<channel>[/<editions>]` will publish built images into the release channel.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
  - `editions` is a comma-separated list of editions `ce | ee | fe | be | se`
- `/suspend/<channel>` will suspend released version.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
- `/e2e/run/<provider> git_ref` will run e2e using provider and an `install` image built from git_ref.
  - `provider` is one of `aws | azure | eks | gcp | openstack | static | vcd | vsphere | yandex-cloud`
  - `git_ref` is a tag `v1.65.2 | test-v1.65.2*` or a branch `main | release-1.65`
- `/e2e/run/<provider> git_ref_1 git_ref_2` will run e2e using provider and git_ref_1 and then switch to git_ref_2.
  - `provider` is one of `aws | azure | eks | gcp | openstack | static | vcd | vsphere | yandex-cloud`
  - `git_ref_1` is a release-* or main branch
  - `git_ref_2` is a release-* or main branch
- `/e2e/use/k8s/<version>` specifies which Kubernetes version to use for e2e test.
  - `version` is one of `1.26 | 1.27 | 1.28 | 1.29 | 1.30 | 1.31 | Automatic`
- `/build git_ref` will run build for release related refs.
  - `git_ref` is a tag `v1.65.2 | test-v1.65.2*` or a branch `main | release-1.65`

**Note 1:**
A single command `/e2e/run/<provider>` will run e2e with default CRI 'containerd' and Kubernetes version '1.27'.
Put `/e2e/use` options below `/e2e/run` command to set specific CRI and Kubernetes version. E.g.:

```
/e2e/run/aws main
/e2e/use/k8s/1.27
/e2e/use/k8s/1.29

This comment will run 2 e2e jobs on AWS with containerd
and with Kubernetes version 1.20 and 1.23 using image built from main branch.
```

```
/e2e/run/aws release-1.35 release-1.36
/e2e/use/k8s/1.27

This comment will create cluster in AWS using Deckhouse built from release-1.35 branch
and then switch to images built from release-1.36 branch.
'use' options are supported, cluster will use containerd and Kubernetes version 1.23.
```

**Note 2:**
'deploy', 'suspend' and 'e2e' commands should run after 'build FE' job is finished.

**Note 3:**
No autobuild for release branch. Run this command after cherry-picking into release branch:
```
/build release-1.65
```

</details>

**Comment by deckhouse-BOaTswain** [comment]
New commit [33f162](https://github.com/deckhouse/deckhouse/commit/33f162e7418008cd1a7748d23e408c0ca25a6930) in branch **main**:
```
Re-generate changelog v1.65.1 (#10254)

Signed-off-by: deckhouse-BOaTswain <89150800+deckhouse-boatswain@users.noreply.github.com>
Co-authored-by: Taior <Taior@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [5606c0](https://github.com/deckhouse/deckhouse/commit/5606c077badf63bfa60b9bdf090f0734f9861cc8) in branch **main**:
```
[user-authz] improve rbacv2 hook (#10241)

Signed-off-by: Stepan Paksashvili <stepan.paksashvili@flant.com>
Signed-off-by: Zhbert <konstantin.nezhbert@flant.com>
Signed-off-by: Kirill Kononovich <41591254+kirkonru@users.noreply.github.com>
Signed-off-by: Artem Kladov <artem.kladov@flant.com>
Co-authored-by: Zhbert <konstantin.nezhbert@flant.com>
Co-authored-by: VV-corrector <vladimir.vodyanitsky@flant.com>
Co-authored-by: Kirill Kononovich <41591254+kirkonru@users.noreply.github.com>
Co-authored-by: Artem Kladov <artem.kladov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [1a4cc4](https://github.com/deckhouse/deckhouse/commit/1a4cc42fd0b4539cc186686d990629ec1d7cfd2e) in branch **main**:
```
Re-generate changelog v1.65.1 (#10399)

Signed-off-by: deckhouse-BOaTswain <89150800+deckhouse-boatswain@users.noreply.github.com>
Co-authored-by: Taior <Taior@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [6a2527](https://github.com/deckhouse/deckhouse/commit/6a2527539808b2ab886d98dfa8b65aa2d51d5be4) in branch **main**:
```
 [docs][control-plane-manager] Minor documentation updates (#10393)

Signed-off-by: Zhbert <konstantin.nezhbert@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [fa1b61](https://github.com/deckhouse/deckhouse/commit/fa1b61f18dc62cd4c5af3e4b036ff323139bd9d1) in branch **main**:
```
[docs] Update information about cluster scaling (#10405)

Signed-off-by: Zhbert <konstantin.nezhbert@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [4cf4f0](https://github.com/deckhouse/deckhouse/commit/4cf4f01216499a76a22dc6ead4005213c82ccdd8) in branch **main**:
```
[user-authz] fix lint (#10411)

Signed-off-by: Pavel Okhlopkov <pavel.okhlopkov@flant.com>
Co-authored-by: Pavel Okhlopkov <pavel.okhlopkov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [d92072](https://github.com/deckhouse/deckhouse/commit/d920728ed79531cdd3b5c4d4d4fd8e0dbab13658) in branch **main**:
```
[docs][candi] update config references (#10408)

Signed-off-by: Emmanuel Ferdman <emmanuelferdman@gmail.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [216ce3](https://github.com/deckhouse/deckhouse/commit/216ce3d7037be5c9ef50e9235eb88e43707accd4) in branch **main**:
```
[docs] Fix links (#10375)

Signed-off-by: l.lysenko <l.lysenko@vk.team>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [d0c4f8](https://github.com/deckhouse/deckhouse/commit/d0c4f87671084a49475c939262546867a7da9e6b) in branch **main**:
```
fix ingress validation (#10407)

Signed-off-by: Pavel Okhlopkov <pavel.okhlopkov@flant.com>
Co-authored-by: Pavel Okhlopkov <pavel.okhlopkov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [8b6944](https://github.com/deckhouse/deckhouse/commit/8b6944760f2a00aadb5b18753f3a5af675c3faba) in branch **main**:
```
Update CODEOWNERS (#10395)

Signed-off-by: Ruslan Abdullaev <139212097+raabdullaev@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [7dda91](https://github.com/deckhouse/deckhouse/commit/7dda91f3efaf389e12559c2ea9a9b5d507c7102a) in branch **main**:
```
Re-generate changelog v1.63.12 (#10400)

Signed-off-by: deckhouse-BOaTswain <89150800+deckhouse-boatswain@users.noreply.github.com>
Co-authored-by: deckhouse-BOaTswain <deckhouse-BOaTswain@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [6cc8b8](https://github.com/deckhouse/deckhouse/commit/6cc8b8d97b705c03be08ffe72933ce3afd194e86) in branch **main**:
```
[candi] Fix failed step reboot by bootstrap (#10417)

Signed-off-by: Dmitrii Kovalkov <dmitrii.kovalkov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [688ac6](https://github.com/deckhouse/deckhouse/commit/688ac6231c96c661cba089d5beff68e816d28fd1) in branch **main**:
```
[user-authn] fix: numbers in dex groups (#10211)

Signed-off-by: Evsyukov Denis <denis.evsyukov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New tag **v1.65.2** is created.
